app/workers/education_form/education_facility.rb

```diff
@@ -84,6 +82,24 @@ def self.region_for(model)
         DEFAULT
       end
     end
+
+    # Claims are sent to different RPOs based first on the location of the school
+    # that the claim is relating to (either `school` or `newSchool` in our submissions)
+    # or to the applicant's address (either as a relative or the veteran themselves)
+    # rubocop:disable Metrics/CyclomaticComplexity
+    def self.routing_address(record, form_type:)
+      case form_type.upcase
+      when '1990'
+        record.school&.address || record.veteranAddress
+      when '1990N'
+        record.educationProgram&.address || record.veteranAddress
+      when '1990E', '5490'
+        record.educationProgram&.address || record.relativeAddress
+      when '1995', '5495'
+        record.newSchool&.address || record.relativeAddress
```

Comment:
1995 doesn't have `relativeAddress`
5495 `newSchool` was changed to `educationProgram`